mycroft/client/text/text_client.py

```diff
@@ -171,6 +172,35 @@ def load_settings():
     global max_log_lines
     global show_meter
 
+    config_file = None
+
+    # Old location
+    path = os.path.join(os.path.expanduser("~"), ".mycroft_cli.conf")
+    if os.path.isfile(path):
+        LOG.warning(" ===============================================")
+        LOG.warning(" ==             DEPRECATION WARNING           ==")
+        LOG.warning(" ===============================================")
```

Comment:
i suggest this whole Configuration section is refactored into smaller pieces, see this file https://github.com/HelloChatterbox/HolmesV/commit/2608fe0e236854bfb9d8ea9f141b51623bc47a45#diff-7a26324169efbe9ae270dc54e1e9476fff06521e22b5294f4e55624081b92bbdR186 

(ignore the fact that i have a flag to determine if xdg is enabled or not, that is for backwards compat with mycroft-core, will remove that once this PR is finally merged)